docs/testing.md

```diff
@@ -13,7 +13,7 @@ Whenever you are fixing a bug or adding new functionality to React Native, you s
 * [Android](testing.md#android)
 * [iOS](testing.md#ios)
 * [Apple TV](testing.md#apple-tv)
-* [End-to-end tests](testing.md#end-to-end-tests)
+* [Manual end-to-end tests](testing.md#manual-end-to-end-tests)
```

Comment:
Should there be a link to the new automated section?

Reply:
It's a level 3 heading under iOS, and the other headings under iOS for different test types don't appear in the list of links.

We could potentially bump it up to a level 2 heading if it's important enough. Once Android E2E is working then it definitely makes sense to be at that level.

Thoughts?

Reply:
That makes sense to me. I missed that it is a level 3 heading. It would be great to get this running on Android